components/headerComponents/NearbyActivities.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
```

Comment:
Sorry I forgot to mention earlier, we can remove the scrollview here and use the ListHeaderComponent prop of the FlatList to allow for a sticky header.